README.md

```diff
@@ -63,29 +63,37 @@ species,file,taxonomy,shallow,broad,mode,uniprot,mcl_test
 Entamoeba_histolytica,Entamoeba_histolytica-test-proteome.fasta,Amoebozoa,eukaryota_odb10,eukaryota_odb10,proteins,true,true
 ```
 - "uniprot" column is a true/false specification indicating whether the proteome comes from UniProt (i.e. has UniProt protein accessions that we can use to annotate)
-- mcl_test is a specification of whether this species is to be included in the MCL inflation parameter test-set. These species must have UniProt protein accessions (for COGEQC protein domain score). 
+- mcl_test is a specification of whether this species is to be included in the MCL inflation parameter test-set. These species must have UniProt protein accessions (for COGEQC protein domain score).
 
 4. Create a parameter file that includes all necessary input, output, and parameter specifications - example below:
 ```
 {
-  "input": "/home/ubuntu/environment/github/phylorthology/test/complete_samplesheet.csv",
-  "mcl_test_input": "/home/ubuntu/environment/github/phylorthology/test/mcl_test_samplesheet.csv",
-  "s3_dir": "s3://organism-selection-engines/sequence-data/aa-seqs/phylorthology-test",
-  "outdir": "results-s3-test",
-  "mcl_inflation": [ "0.8", "0.9", "1.0", "1.1", "1.2", "1.3", "1.4", "1.5", "1.6", "1.7" ]
+  "input": "<FILE_PATH_TO_YOUR_SAMPLESHEET>",
+  "outdir": "results",
+  "mcl_inflation": "0.8,0.9,1.0,1.1,1.2,1.3,1.4,1.5,1.6,1.7"
```

Comment:
I think for simplicity the inflation parameters tested here could just be 1.0, 2.0, 3.0, 4.0, and 5.0. As is the above is somewhat excessive. 

But, I didn't realize you could just have all the numbers in a single set of quotes!